rest-service/manager_rest/test/endpoints/test_plugins_updates.py

```diff
@@ -118,12 +118,11 @@ def test_raises_with_nonexisting_blueprint(self):
             self.client.plugins_update.update_plugins("non_existing")
             self.assertEqual(404, e.exception.status_code)
 
-    def test_raises_blueprint_has_no_deployments(self):
+    def test_no_op_blueprint_has_no_deployments(self):
         self.put_file(*self.put_blueprint_args(blueprint_id='hello_world'))
-        with self.assertRaisesRegexp(CloudifyClientError,
-                                     "The blueprint 'hello_world' has no "
-                                     "deployments to update."):
-            self.client.plugins_update.update_plugins("hello_world")
+        plugins_update = self.client.plugins_update.update_plugins(
+            'hello_world')
+        self.assertEqual(plugins_update.state, STATES.NO_CHANGES_REQUIRED)
```

Comment:
Stage shows this state?

Reply:
I don't think there's a UI for this, at least _I_ haven't created anything for it